dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -25,6 +26,122 @@ public static void prepare() throws Exception {
         IntegrationTestInitService.getInstance().init();
     }
 
+    @Test
+    public void test_match_MimeType_all() throws IOException {
+
+        boolean match = MimeTypeUtils.match("*/*", "image/jpeg");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "*/*" appears 4 times in this file; the first occurrence is on line 32](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5028307)